spec/clang_spec.lua

```diff
@@ -178,4 +177,237 @@ describe("cursor kind", function()
                 end)
                 assert.are.same(expected, children)     
         end)
+end)
+
+function get_last_child(cursor)
+        local last_child
+        cursor:visitChildren(function(cursor, parent)
+                last_child = cursor
+                return "continue"
+        end)
+        return last_child
+end
+
+describe("type object creation", function() 
+        it("succeeds in creating type object", function() 
```

Comment:
We could suppress the `succeeds in` to be more concise:
```suggestion
        it("creates a type object", function() 
```
Please fix all.